OneCommunityGlobal/HighestGoodNetworkApp
# Lucas optimize resolve task dismiss notification

This PR resolves: (PRIORITY LOW) Jae:  Make resolving a task only refresh that person or not refresh the Tasks tab at all. (WIP Lucas) 
Right now when I resolve a task (green check), it refreshes the whole tasks tab. This slows me down a lot because I’m often resolving many tasks at once. I’d like it to just make the task disappear from that person without refreshing and reloading the whole tab
Do the same for when resolving a red bell as per this

### MAIN CHANGES ###

Previously, when a member attempted to resolve a task, the tasks component would reload, posing challenges in managing multiple tasks. However, an improvement has been implemented: now, when resolving a task, the component no longer undergoes a complete reload. In the past, an issue arose where resolving a task would result in a blank screen, failing to successfully complete the task. To tackle this problem, the visual aspect has been enhanced to ensure effective task resolution and prevent any disruption.

### HOW TO TEST ###

- checkout the current branch
- npm install
- npm run start:local
- create a task and assign it to an user
- try to resolve the task clicking the green icon next to it
- Wait until the modal loads the task, confirm it and wait until the modal is closed
- see if the task is gone
- open anonymous tab and loggon with the same user that dismissed the task
- see if the task is gone

### READ THIS, IT'S VERY IMPORTANT!!!!!!!!!!!!! ###

On the TeamMemberTasks I've stopped putting tasks on the "tasks" state, because I think it was not needed on any part of the component. However, please investigate to see if it's not needed. I've already done some investigation and found out that it in fact does nothing, help me out with this!
